File: graal-java-graphql/src/main/java/lan/test/graal/HelloController.java
package lan.test.graal;

import io.micronaut.http.MediaType;
import io.micronaut.http.annotation.Controller;
import io.micronaut.http.annotation.Get;

@Controller("/hello")
public class HelloController {

    @Get(produces = {MediaType.TEXT_JSON})
    public Type index() {
        return Type.DEFAULT_TYPE;
    }
}

File: graal-kotlin-graphql/src/main/kotlin/lan/test/graal/HelloController.kt
package lan.test.graal

import io.micronaut.http.MediaType
import io.micronaut.http.annotation.Controller
import io.micronaut.http.annotation.Get

@Controller("/hello")
class HelloController {
    @Get(produces = [MediaTyspe.TEXT_JSON])
    fun index(): Type {
        return Type.DEFAULT_TYPE
    }
}